ArmTemplates/azure-search.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "azureSearchName": {
+            "type": "string",
+            "minLength": 2,
+            "maxLength": 60,
+            "metadata": {
+                "description": "Service name must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and is limited between 2 and 60 characters in length."
+            }
+        },
+        "azureSearchSku": {
+            "type": "string",
+            "defaultValue": "standard",

```

Comment:
I think default should be basic.  We've load tested at 20 searches per second on basic.  That said you can't scale from basic to standard which is an argument to default to standard.  If we're creating lots of small search services basic will be a lot more cost effective ~£50 against ~£180 per month

Reply:
Agreed and changed